librz/core/devirtualize_cxx.c

```diff
@@ -0,0 +1,234 @@
+// SPDX-FileCopyrightText: 2025 tushar3q34 <tushar3q34@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_util.h>
+#include <rz_analysis.h>
+#include <rz_core.h>
+#include "core_private.h"
+
+/**
+ * `RZ_TAINT_MODE_OFF`   : No tainting
+ * `RZ_TAINT_MODE_RED`   : `new` identified, taint till constructor
+ * `RZ_TAINT_MODE_BLUE`  : constructor identified, taint till var write
+ */
+typedef enum {
+	RZ_TAINT_MODE_OFF,
+	RZ_TAINT_MODE_RED,
+	RZ_TAINT_MODE_BLUE,
```

Comment:
```suggestion
	RZ_TAINT_MODE_OFF, ///< No tainting
	RZ_TAINT_MODE_RED, ///< `new` identified, taint till constructor - Be a little more descriptive here. Or describe the algorithm in more detail somewhere else and refer to it.
	RZ_TAINT_MODE_BLUE, ///< Constructor identified, taint until variable write - Be more descriptive as well, as above.
```

Better to be consistent in the whole code base.

Also, please write out the sentences with proper grammar and all. Using slang or abbreviations can be hard for people with English as their second language.

For the details, you can also describe the meaning of the colors in more detail by describing the enum with `\brief `